hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -591,6 +591,8 @@ public static PropertyBuilder withPropertyBuilder() {
 
     private HoodieTableType tableType;
     private String tableName;
+    private String tableSchema;
```

Comment:
just calling out that these fields are only set and used by the sql tables. if someone has an existing Hudi table, that they want to operate using SQL now. Do we need some code in `UpgradeDowngrade` ? Something to think about and at-least file a new JIRA for.